### PR TITLE
add macOS and windows installer

### PR DESCRIPTION
**Pull request by rejbasket**
This branch will introduce a macOS installer and a windows installer on top of the inx-translations branch.

### CODE REVIEW

File: installer_scripts/scripts/postinstall [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+sudo mv /tmp/inkstitch/ $HOME/Library/Application\ Support/org.inkscape.Inkscape/config/inkscape/extensions/
```
Comment (lexelby) [inline installer_scripts/scripts/postinstall:3]: I know very little about macos, but I'm curious... why does this require sudo?

**Comment by lexelby** [comment]
Thank you so much for doing this, it's going to be _huge_ for our users to have an installer.

Does this do the code signing for MacOS?  And do we need to sign for Windows?

**Comment by lexelby** [comment]
@rejbasket Oh no, I'm sorry!  I think I cancelled one of your builds.

**Comment by lexelby** [comment]
I don't know anything about macs, but ugo+rwx seems risky.  Any other user on the same mac could modify the files... Can we instead chown the files to the user?

**Comment by kaalleen** [comment]
Notarizing dev builds can be temporarily be enabled by setting `NOTARIZE_DEVELOPMENT_BUILDS` in build-stribution-archives to true

**Review by kaalleen — APPROVED** [review]

**Pull request merged**